docs/change-log/index.md

```diff
@@ -13,16 +14,183 @@
+- httpdb runtime-related APIs using the deprecated runtimes REST APIs, for example: `delete_runtime_object`
+
+### Breaking changes
+
+- The behavior of ingest with aggregation changed in v1.3.0. Now, when you ingest a "timestamp" column, it returns <br>
+`<class 'pandas._libs.tslibs.timestamps.Timestamp'>`. <br>Previously, it returned `<class 'str'>`
+
+- Any target data that was saved using Redis as an online target with storey engine (RedisNoSql target, introduced in 1.2.1) is not accessible after upgrading to v1.3. (Data ingested subsequent to the upgrade is unaffacted.)
+
+### Platform limitation
+
+When running v1.3.0 with a platform version < v3.5.3, you must upgrade pyopenssl to 22.1.0 or higher.
+Use this flow:
+```
+{{python3 -m pip install --upgrade pyopenssl>=23
+$CONDA_HOME/bin/conda update -y conda
+$CONDA_HOME/bin/conda install -y pip
+./align_mlrun.sh}}
+```
+
+### Closed issues
+
+- Fix: Project page displayed an empty list after an upgrade [View in Git](https://github.com/mlrun/ui/pull/1611).
+- Jobs and Workflows pages now display the tag of the executed job (as defined in the API). [View in Git](https://github.com/mlrun/ui/pull/1632).
+- Users with developer and data permissions can now add members to projects they created. (Previously appeared successful in the UI but users were not added). [View in Git](https://github.com/mlrun/ui/pull/1617).
+- Error on Spark ingestion with offline target without defined path (error: `NoneType` object has no attribute `startswith`). Fix: default path defined. [View in Git](https://github.com/mlrun/mlrun/pull/3118).
+- `add_aggregation` over Spark fails with `AttributeError` for sqr and stdvar. [View in Git](https://github.com/mlrun/mlrun/pull/3131).
+- Fix: UI Projects' metrics show N/A for all projects when ml-pipeline is down. [View in Git](https://github.com/mlrun/ui/pull/1613).
+- `project.list_models()` did not function as expected for tags and labels. The `list_artifacts` method now accept a dictionary, and 
+    docstrings were added for httpdb and for MLRunProject methods: both `list_artifacts` and `list_models`. [View in Git](https://github.com/mlrun/mlrun/pull/2988).
+- Fix: Failed MLRun Nuclio deploy needs better error messages.     
+- Fix: Second call to Slack notifier with same webhook does not add another notifier. Resolved by the future (not yet released) notification mechanism. 
+- Fixed the Dask Worker Memory Limit Argument. [View in Git](https://github.com/mlrun/mlrun/pull/3123).
+- Aggregation over a fixed-window that starts at or near the epoch now functions as expected. [View in Git](https://github.com/mlrun/storey/pull/418).
+- Fixed model-monitoring incompatibility issue with mlrun client running v1.1.x and a server running v1.2.x. [View in Git](https://github.com/mlrun/mlrun/pull/3180).
+
+    
+    
+### See more
+- [MLRun change log in GitHub](https://github.com/mlrun/mlrun/releases/tag/v1.3.0)
```

Comment:
I think we want to have the exact change log above in the `tag/v1.3.0` so no need to point to there

Reply:
moved to UI heading in New and updated features

Reply:
moved under UI heading